app/models/physical_ticket.rb

```diff
@@ -4,4 +4,12 @@ class PhysicalTicket < ActiveRecord::Base
   has_one :conference, through: :ticket_purchase
   has_one :user, through: :ticket_purchase
   has_many :ticket_scannings
+
+  after_create :send_ticket_confirmation_mail
+
+  def send_ticket_confirmation_mail
+    if conference.email_settings.send_on_ticket_confirmation?
+      Mailbot.ticket_confirmation_mail(conference, user, self).deliver_later
```

Comment:
No unfortunately we cannot send the ticket as a attachment in stripe confirmation email. Customisation of stripe confirmation email is only limited to changing the organisation name, logo.